rllib/tests/git_bisect/debug_learning_failure_git_bisect.py

```diff
@@ -159,10 +159,13 @@
         os.chdir("../")
 
     try:
-        subprocess.run("ray start --head --include-dashboard false".split(" "))
+        subprocess.run("ray start --head".split(" "))
     except Exception:
         subprocess.run("ray stop".split(" "))
-        subprocess.run("ray start --head --include-dashboard false".split(" "))
+        try:
+            subprocess.run("ray start --head".split(" "))
+        except Exception as e:
+            print(f"ERROR: {e.args[0]}")
```

Comment:
What's the reason for catching this error again? Maybe consider a loop with try-except if multiple retries are necessary.

Reply:
It's just in case of ray already running. Then this would fail and we should - ideally - stop and re-start just in case an older version of ray is running (we do a recompile from source before this so it's essential this is the newly built version).